test/utils.js

```diff
@@ -1,8 +1,13 @@
 'use strict';
 const assert = chai.assert;
 
-export function almostEqual(
-    a, b, episilon = 1e-6, rtol = 5.0 * 1.1920928955078125e-7) {
+// The following 4 constants were used for converted tests from NNAPI CTS
+export const atol = 1e-5;
+export const rtol = 5.0 * 1.1920928955078125e-7;
+export const atolRelaxed = 5.0 * 0.0009765625;
+export const rtolRelaxed = 5.0 * 0.0009765625;
```

Comment:
It would be good to define an AccuracyCriterion class and create different criterias. For example,
```js
// define
const opFp32AccuracyCriteria  = new AccuracyCriterion(1e-6, 5.0 * 1.1920928955078125e-7);
const ctsFp32RestrictAccuracyCriteria = new AccuracyCriterion(1e-5, 5.0 * 1.1920928955078125e-7);
const ctsFp32RelaxedAccuracyCriteria = new AccuracyCriterion(5.0 * 0.0009765625, 5.0 * 0.0009765625);

// use atol and rtol
console.log(opFp32AccuracyCriteria.atol, opFp32AccuracyCriteria.rtol);

// check value
checkValue(output, expected, ctsFp32RestrictAccuracyCriteria);
```

What do you think?

Reply:
@huningxin  Thanks your suggestion, I updated it. PTAL, thanks.